health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/timelinePopUpWrapper.js

```
@@ -1,746 +1,38 @@
-                "rating": 0,
-                "escalated": false
-            }
-        ],
-        "totalCount": 0
-    }
     // Manage timeline data
     const [timelineSteps, setTimelineSteps] = useState([]);
 
     useEffect(() => {
-        if (hardCodeData && hardCodeData.ProcessInstances) {
+        if (workflowData && workflowData.ProcessInstances) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)